src/main/java/com/oltpbenchmark/benchmarks/resourcestresser/procedures/CPU1.java

```diff
@@ -36,7 +38,7 @@ public class CPU1 extends Procedure {
         for (int i = 1; i <= ResourceStresserWorker.CPU1_nestedLevel; ++i) {
             complexClause = "md5(concat(" + complexClause + ",?))";
         }
-        cpuSelect = new SQLStmt("SELECT count(*) FROM (SELECT " + complexClause + " FROM " + ResourceStresserConstants.TABLENAME_CPUTABLE + " WHERE empid >= 0 AND empid < 100) AS T1");
+        cpuSelect = new SQLStmt("SELECT count(*) FROM (SELECT " + complexClause + " FROM " + ResourceStresserConstants.TABLENAME_CPUTABLE + " WHERE empid >= 0 AND empid < 100) T1");
```

Comment:
Is this cross DBType compatible?

Reply:
It should be for all DB type (for example [spanner](https://cloud.google.com/spanner/docs/reference/standard-sql/query-syntax#from_clause))

Oracle doesn't work if AS is kept, and it's not possible to use dialect file for these particular statements.

Reply:
We could certainly add switch statements on DBType if necessary.  Let's check the CI results, but as long as this works across all currently supported DBs I'm fine with the change as is.